tests/force_insert_update/tests.py

```diff
@@ -66,3 +77,153 @@ def test_force_update_on_inherited_model_without_fields(self):
         a.save()
         a.value = 2
         a.save(force_update=True)
+
+
+class ForceInsertInheritanceTests(TestCase):
+    def test_force_insert_on_grandparent_parent_doesnot_exists(self):
+        with self.assertNumQueries(3):
+            Child(id=1, name="Child-1", value=10).save(force_insert=(models.Model,))
```

Comment:
This is not resolved.

Reply:
I think the idea is that from a third-party app perspective it might be useful to force the insertion of all bases of a Model subclasses independently  of its base chain.

In a certain sense `force_insert=(models.Model,)` is the initially requested feature of ticket-30382 to have `force_insert=True` force the insertion of all bases but since it would be backward incompatible to do so we document another way to achieve it.

Reply:
And also in the current version of `_validate_force_insert()` it does not take any extra checks. So not supporting this will take extra checks now.